app/containers/AudioPlayer/Seek.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { LayoutChangeEvent, View, TextInput } from 'react-native';
+import { PanGestureHandler } from 'react-native-gesture-handler';
+import Animated, {
+	SharedValue,
+	runOnJS,
+	useAnimatedGestureHandler,
+	useAnimatedProps,
+	useAnimatedStyle,
+	useDerivedValue,
+	useSharedValue
+} from 'react-native-reanimated';
+
+import styles from './styles';
+import { useTheme } from '../../theme';
+
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
+
+interface ISeek {
+	duration: SharedValue<number>;
+	currentTime: SharedValue<number>;
+	loaded: boolean;
+	onChangeTime: (time: number) => Promise<void>;
+}
+
+const BUTTON_HIT_SLOP = {
+	top: 8,
+	right: 8,
+	bottom: 8,
+	left: 8
+};
+
+const Seek = ({ currentTime, duration, loaded = false, onChangeTime }: ISeek) => {
+	const { colors } = useTheme();
+
+	const maxWidth = useSharedValue(1);
+	const x = useSharedValue(0);
+	const current = useSharedValue('00:00');
+	const scale = useSharedValue(1);
+	const isHandlePan = useSharedValue(false);
+	const onEndGestureHandler = useSharedValue(false);
+	const isTimeChanged = useSharedValue(false);
+
+	const styleLine = useAnimatedStyle(() => ({
+		width: x.value,
+		zIndex: 2
+	}));
+
+	const styleThumb = useAnimatedStyle(() => ({
+		transform: [{ translateX: x.value }, { scale: scale.value }]
+	}));
+
+	const onLayout = (event: LayoutChangeEvent) => {
+		const { width } = event.nativeEvent.layout;
+		maxWidth.value = width - 12;
+	};
+
+	const gestureHandler = useAnimatedGestureHandler({
```

Comment:
Matches the name of the function.

```suggestion
	const onGestureEvent = useAnimatedGestureHandler({
```